testing/src/main/java/org/oppia/android/testing/RecyclerViewMatcher.kt

```diff
@@ -1,12 +1,15 @@
-package org.oppia.android.app.recyclerview
+package org.oppia.android.testing
```

Comment:
We ought to have a subpackage for this.

@fsharpasharp or @anandwana001 do you have any thoughts on how we should organize where Espresso helpers go in the testing package?

Reply:
in one of the PR we are doing `org.oppia.android.testing.robolectric` so we can go with `org.oppia.android.testing.espresso`

Reply:
That sounds good to me. WDYT @Sparsh1212?